src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -108,10 +119,12 @@ CpsRelatedContent.propTypes = {
   // Both pages use CPS, so the data schema is the same
   // This can be found under CPS ARES payloads: relatedContent.groups[0].promos
   content: arrayOf(shape(storyItem)),
+  enableGridWrapper: bool,
 };
 
 CpsRelatedContent.defaultProps = {
   content: [],
+  enableGridWrapper: false,
```

Comment:
I feel like we might at some point need to start decorating props with JSdocs just so we don't forget what is it they're actually doing